target/linux/ramips/dts/mt7621_mercury_km08-708h.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2023, Muhammad Al-Qady <m.ismael@gmail.com>
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "mercury,km08-708h", "mediatek,mt7621-soc";
+	model = "Mercury KM08-708H";
+
+	aliases {
+		label-mac-device = &wan;
+		led-boot = &led_wlan;
+		led-failsafe = &led_wlan;
+		led-upgrade = &led_wlan;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+		bootargs-override = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_wlan: wlan {
+			label = "green_wlan";
```

Comment:
The label is not needed any more, it will automatically generate the `green:wlan` label. Same for the other LED.

Reply:
this apply to v24.10.2 ? because i am testing on it.